workspaces/ui-v2/src/optic-components/common/EditableTextField.tsx

```diff
@@ -16,6 +16,7 @@ type EditableTextFieldProps = {
 export enum TextFieldVariant {
   SMALL,
   REGULAR,
+  FIELDORPARAM,
```

Comment:
This is really hard coded but I think this is a start to consolidate the usages of textfields - ideally we have a somewhat consistent style of input field that can be adapted to different containers.

I think these text fields are _basically_ identical apart from a couple of interactions / styles - we can consolidate them later though